README.md

```diff
@@ -29,6 +29,7 @@ If you run into any issues while working in this repo, please check out [the tro
 - `npm run build` - builds the UI
 - `npm run clean` - deletes the build/generated content directories
 - `npm run lint` - lints the codebase. See [`Linting`](./docs/Linting.md) for the individual linting steps
+- `npm run storybook` - runs [Storybook](./docs/Architecture.md#storybook) for the UI components.
```

Comment:
Could a link also be added to https://github.com/strimzi/strimzi-ui/blob/master/docs/README.md ?

Reply:
https://github.com/strimzi/strimzi-ui/blob/master/docs/README.md already has a link to the Architecture.md doc?

Reply:
Beg pardon - right you are - sorry!